tutorials/3d/csg_tools.rst

```diff
@@ -296,8 +296,39 @@ in the **Scale** property will cause the texture to repeat more often.
          the material onto, click the dropdown arrow next to its material
          property then choose **Paste**.
 
+.. _doc_csg_tools_converting_to_mesh_instance_3d:
+
+Converting to MeshInstance3D
+----------------------------
+
+Since Godot 4.4, you can convert a CSG node and its children to a :ref:`class_MeshInstance3D` node.
+
+This has several benefits:
+
+- Bake lightmaps, since UV2 can be generated on a MeshInstance3D.
+- Bake occlusion culling, since the occlusion culling bake process only takes MeshInstance3D into account.
+- Faster loading times, since the CSG mesh no longer needs to be rebuilt when the scene loads.
+- Better performance when updating the node's transform if using the mesh within another CSG node.
+
+To convert a CSG node to a MeshInstance3D node, select it, then choose
+**CSG > Bake Mesh Instance** at the top of the 3D editor viewport. The MeshInstance3D node
```

Comment:
```suggestion
**CSG > Bake Mesh Instance** in the toolbar. The MeshInstance3D node
```
See [Common Vocabulary](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#common-vocabulary-to-use-in-godot-s-documentation). "Toolbar" is still used though it seems this image needs an update for Godot 4